Nighttime-Operations/Observatory/DIMM/index.rst

```diff
@@ -4,11 +4,52 @@
 DIMM
 ###############
 
-This page contains some common calibrations procedures.
+The differential image motion monitor (DIMM) is a standard instrument to measure astronomical "seeing", or atmospheric turbulence strength. 
+
+The DIMM principle is to produce "twin images" of a bright star by a mask with two apertures and a wedge prism at the entrance pupil of the telescope. 
+The rms of the fluctuations of the angular distance between the two images is proportional to the atmospheric optical turbulence, that in turn, allows to predict the seeing FWHM. 
```

Comment:
Consider:
... that in turn enables a prediction of the expected FWHM in long exposures.

Reply:
👍